specs/resolvers/cluster-resolvers.spec

```diff
@@ -0,0 +1,59 @@
+PIPELINES-23
+# Cluster resolvers spec
+
+## Checking the functionality of cluster resolvers#1: PIPELINES-23-TC01
+Tags: e2e, sanity
+Component: Resolvers
+Level: Integration
+Type: Functional
+Importance: High
+
+Steps:
+    * Create project "resolver-tasks"
+    * Create
+      |S.NO|resource_dir                                                |
+      |----|------------------------------------------------------------|
+      |1   |testdata/resolvers/tasks/resolver-task2.yaml                |
+    * Create project "resolver-pipelines"
+    * Create
+      |S.NO|resource_dir                                                |
+      |----|------------------------------------------------------------|
+      |1   |testdata/resolvers/pipelines/resolver-pipeline.yaml         |
+    * Create project "resolver-pipelineruns"
```

Comment:
IMHO we could prefix all 3 projects with `releasetest-` so that it's named similarly to automatically generated namespaces

Reply:
Please add step `* Verify ServiceAccount "pipeline" exist` after creating this project. Sometimes clusters are slow and SA is created only after test creates a pipeline run